docs/en/multi_modal/cogvlm.md

```diff
@@ -0,0 +1,35 @@
+# cogvlm
+
+## Introduction
+
+CogVLM is a powerful open-source visual language model (VLM). LMDeploy supports CogVLM-17B models like [THUDM/cogvlm-chat-hf](https://huggingface.co/THUDM/cogvlm-chat-hf) and CogVLM2-19B models like [THUDM/cogvlm2-llama3-chat-19B](https://huggingface.co/THUDM/cogvlm2-llama3-chat-19B) in PyTorch engine.
+
+## Quick Start
+
+### Prepare model
+
+Download CogVLM models using huggingface-cli.
+
+```shell
+# install huggingface-cli
+pip install -U "huggingface_hub[cli]"
```

Comment:
We don't need to do `pip install` here, since `huggingface-cli` is installed once `transformers` package is installed.